cmake/CeleritasUtils.cmake

```
@@ -64,6 +64,39 @@ CMake configuration utility functions for Celeritas.
   The ``<input>`` must be a relative path to the current source directory, and
   the ``<output>` path is configured to the project build "include" directory.
 
+.. command:: celeritas_setup_option
+
+  Add a single compile time option value to the list::
+
+    celeritas_setup_option(<var> <option> [conditional])
+
+  This appends ``<option>`` to the list ``<var>_OPTIONS`` if ``${conditional}``
+  is true or the argument is not provided; and to ``<var>_DISABLED_OPTIONS`` if
+  the variable is present and false.
+
+.. command:: celeritas_define_options
+
+  Set up CMake variables for defining configure-time option switches::
+
+    celeritas_define_options(<var> <doc>)
+
+  This will set <var> to the first item of the list ``${<var>_OPTIONS}``. It
+  will validate that the selection is one of the list and default to the first
+  item in the list.
```

```suggestion
  This will set <var> to the first item of the list ``${<var>_OPTIONS}``.
```
Aren't the next two sentences 'redundant' (Since it is the first element in the list it is guaranteed to be in the list)

---

Reading the actual code.  The additional sentences are not redundant, just confusing.
```suggestion
  If <var> is not yet set, this will set it to the first item of the list ``${<var>_OPTIONS}``. 
  Otherwise It will validate that the pre-existing selection is one of the list.
```